### PR TITLE
React cli convention

### PR DESCRIPTION
**Pull request by liruochen1998**
I think this is a good convention to follow. Using react-cli to create components.

### CODE REVIEW

File: src/components/TogetherButton/TogetherButton.stories.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import TogetherButton from './TogetherButton';
+
+storiesOf('TogetherButton', module).add('default', () => <TogetherButton />);
```
Comment (Jaega) [inline src/components/TogetherButton/TogetherButton.stories.js]: Can you explain a little bit what a story is used for?

Comment (liruochen1998) [reply src/components/TogetherButton/TogetherButton.stories.js]: That's a good point, I wasn't paying close attention to this when I set it up. And I just googled it seems that it's about some project-independent testing. I think we can get rid of this. I'll recommit

Comment (liruochen1998) [reply src/components/TogetherButton/TogetherButton.stories.js]: It will probably be some learning curve so I don't think we need this for now.

**Review by carolinepar — APPROVED** [review]

**Review by Jaega — APPROVED** [review]
Looks good!

**Pull request merged**